andela/helios-ah-backend
# #161779769 Configure eslint for the application

#### What does this this PR do?
This PR configures eslint for the application using the airbnb style guide.

#### Description of Tasks Completed
- Install eslint, eslint-config-airbnb, eslint-plugin-jsx-a11y, eslint-plugin-react, eslint-plugin-import
- Add .eslintrc.json file
- Edit .eslintrc.json file based on airbnb style guide

#### How should this be manually tested?
- Create a file with .js extension
- Try using var for declaring a variable
- Eslint should flag it.

#### Background Context Provided
- N/A

#### What are the Relevant Pivotal Tracker Stories?
[#161779769](https://www.pivotaltracker.com/n/projects/2223726)
